src/main/webapp/i18n/de/student-dashboard.json

```
@@ -4,6 +4,7 @@
             "title": "Deine aktuellen Kurse",
             "recentlyAccessed": "Zuletzt besuchte Kurse",
             "otherCourses": "Andere Kurse",
+            "noCourses": "Du bist in keinem Kurs eingeschrieben. Schreibe dich manuell ein oder kontaktiere die Kursadministration.",
```

```suggestion
            "noCourses": "Du bist in keinem Kurs eingeschrieben. Schreibe dich manuell ein oder kontaktiere die Kurslehrende.",
```
or
```suggestion
            "noCourses": "Du bist in keinem Kurs eingeschrieben. Schreibe dich manuell ein oder kontaktiere die Kurslehrkraft.",
```
to keep in line with our other translations